setup.cfg

```
@@ -16,7 +16,7 @@ count = True
 ; D401 First line should be in imperative mood; try rephrasing
 ; W503: line break before binary operator (This is incompartible with Black,
 ;       See: https://black.readthedocs.io/en/stable/the_black_code_style.html#line-breaks-binary-operators)
-ignore = D401,W503
+ignore = D401,W503,E402
```

Some info regarding the error we are ignoring.
```suggestion
; E402: Import at top, too many circular import fixes.
ignore = D401,W503,E402
```